client/src/App.tsx

```diff
@@ -9,13 +9,11 @@ import {
   AuditAdminView,
   JurisdictionAdminView,
 } from './components/MultiJurisdictionAudit'
```

Comment:
might be something to do in a later PR, but now that we don't have a SingleJurisdictionAudit, we don't really need the separate directory for MultiJurisdictionAudit components.

i think it might be clearer to organize the components by user type, since they are basically separate apps: AuditAdmin, JurisdictionAdmin, and AuditBoard

Reply:
@jonahkagan Oh, I really like that idea. Yeah definitely future PR because that'd be a significant refactor possibly, though.